CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 
 option(TARADINO_SHAREWARE "Build shareware version" OFF)
 option(TARADINO_WERROR "Enable -Werror on build" OFF)
+option(TARADINO_INSTALL "Enable install target" OFF)
```

Comment:
Thanks for this. However, I don't understand why we should need to make the install target optional?